apps/minaintyg/src/store/slice/session.slice.ts

```diff
@@ -27,9 +27,6 @@ const sessionSlice = createSlice({
     reset() {
       return initialState
     },
-    startSession(state) {
-      state.hasSession = true
-    },
```

Comment:
This was only used for testing, is now replaced with using `getUser` endpoint dispatch